<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upgrade_menu</class>
 <widget class="QDialog" name="upgrade_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extended Upgrade Settings</string>
  </property>
  <widget class="QLabel" name="initialLevelPlaceholder">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxLevelPlaceholder">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Level:</string>
   </property>
  </widget>
  <widget class="QLabel" name="upgradeNameHeaderPlaceholder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upgrade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="upgradeName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="initialLevelTextbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="maxLevelTextbox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="escalationHeader">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Escalations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lowerEscalationButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="upperEscalationButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
